utility/src/main/java/org/oppia/util/logging/ExceptionsHelper.kt

```diff
@@ -0,0 +1,47 @@
+package org.oppia.util.logging
+
+import org.oppia.app.model.ExceptionLog
+
+/** Helper class for conversion of exceptionLog proto object to an Exception. */
+class ExceptionsHelper {
```

Comment:
Could we create an extension function on ``ExceptionLog``, instead? E.g. ``ExceptionLog.toException``? That might be cleaner than having a generic helper/utility for random exception things.

Reply:
Yes, it's a lot cleaner now. Thanks !